sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/RemoteDependencyData.cs

```diff
@@ -34,6 +34,9 @@ public RemoteDependencyData(int version, Activity activity, ref ActivityTagsProc
                 case OperationType.Messaging:
                     SetMessagingDependencyProperties(activity, ref activityTagsProcessor.MappedTags);
                     break;
+                default:
+                    Target = activityTagsProcessor.MappedTags.GetTargetUsingServerAttributes();
```

Comment:
We don’t need to make it complex. I would just use AzMonList.GetTagValue(ref tagObjects, SemanticConventions.AttributeServerAddress)

Reply:
The spec says that "server.port" also needs to be considered. I found this helper method which handles both.

Reply:
Please update the changelog accordingly.

Reply:
done.